README.md

```diff
@@ -1,128 +1,208 @@
-Доступные настройки браузера:
+`<browser-id>` valus is used for browser identification.
 
-* `desiredCapabilities` (обязательная) - Необходимые для этого браузера WebDriver [DesiredCapabilites](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)
-* `sessionsPerBrowser` - Количество одновременно запущеных сессий для браузера с данным id. По умолчанию 1.
-* `retry` - Сколько раз тест может быть перезапущен. По умолчанию 0.
+Available browser options:
 
-### Подготовка webdriver-сессии к работе
-Подготовка сессии к работе (например, установка специфических для пользователя команд) выполняется в секции `prepareBrowser`.
-Формат секции:
-```js
-prepareBrowser: function(browser) {
-    // do setup here
-}
-```
+Option name               | Description
+------------------------- | -------------
+`desiredCapabilities`     | **Required.** Used WebDriver [DesiredCapabilites](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)
+`sessionsPerBrowser`      | Number of sessions which are run simultaneously. Default value is `1`.
+`retry`                   | How many times test should be rerun. Default value is `0`.
 
-В данную функцию будет передана сессия `webdriver.io`.
+### grid
+Selenium grid URL. Default value is `http://localhost:4444/wd/hub`.
 
-### Кастомизация настройки окружения
+### baseUrl
+Base service url. Default value is `http://localhost`.
 
-Данные в конфиге можно изменять в зависимости от дополнительных условий в функции `prepareEnvironment`. Использование этой функции не обязательно, она для удобства.
+### timeout
+Time for text execution. Default value is `60000` ms.
 
-## Плагины
-hermione поддерживает расширения.
-Плагин представляет из себя модуль, экспортирующий одну единственную функцию. Эта функция принимает на вход два параметра:
-* инстанс hermione
-* опции плагина из конфига
+### waitTimeout
+Time for web page events. Default value is `10000` ms.
 
-В момент запуска тестов плагин будет загружен и вызван с текущим инстансом hermione и опциями конфига, соответствующими данному плагину.
+### slow
+If test execution time greater then this value, then test is slow. Default value is `10000`.
 
-Если название модуля плагина начинается с `hermione-`, то в конфиге этот префикс можно опустить. Если в зависимостях есть два модуля, например `hermione-some-module` и `some-module`, то приоритет отдается модулю с префиксом.
+### debug
+Turn webdriver debug mode on. Default value is `false`.
 
-Пример.
+### sessionsPerBrowser
+Number of sessions which are run simultaneously. Global value for all browsers. Default value is `1`.
+
+### retry
+How many times test should be rerun. Global value for all browsers. Default value is `0`.
+
+### plugins
+`hermione` plugin is a module which exports a single function. The function has two arguments:
+* hermione instance;
+* plugin options from configuration file.
+
+Plugins will be loaded when `hermione` runs tests.
```

Comment:
`loaded before`
otherwise you wouldn't be able to listen to `RUNNER_START` event :)